src/common/components/clusterWizard/networkingSteps/ManagedNetworkingControlGroup.tsx

```diff
@@ -7,19 +7,28 @@ import { useTranslation } from '../../../hooks/use-translation-wrapper';
 export interface ManagedNetworkingControlGroupProps {
   disabled: boolean;
   tooltip?: string;
+  isUmnDisabled?: boolean;
 }
 
 const GROUP_NAME = 'managedNetworkingType';
 export const ManagedNetworkingControlGroup = ({
   disabled = false,
   tooltip,
+  isUmnDisabled = false,
 }: ManagedNetworkingControlGroupProps) => {
   const tooltipProps: TooltipProps = {
     hidden: !tooltip || !disabled,
     content: tooltip,
     position: 'top',
   };
+
   const { t } = useTranslation();
+  const tooltipUmnDisabled = t('ai:User-Managed Networking is not supported when using Nutanix');
```

Comment:
There was another comment related to this text.

Why not pass the tooltip text via something similar to `networkManagementDisabledReason` so that there is no coupling because why the option is disabled and the fact that it is disabled?

Reply:
ok, I'll change it